cs.html

```diff
@@ -961,689 +1180,684 @@ <h2>Constructors</h2>
+            <a>[[\Writable]]</a> slot.</p>
+          </dd>
+          <dt><code>writeBufferedAmount</code> of type <span class="idlAttrType"><a>unsigned
+          long</a></span>, readonly</dt>
+          <dd>
+            <p>The <dfn id="dom-outgoingstream-writable"><code>writeBufferedAmount</code></dfn>
+            attribute represents the number of bytes of application data
+            that have been queued using <code>write</code> but that, as of the last
+            time the event loop started executing a task, had not yet been transmitted
+            to the network. This includes any data sent during the execution of the
+            current task, regardless of whether the <a>user agent</a> is able to
+            transmit text asynchronously with script execution. This does not
+            include framing overhead incurred by the protocol, or buffering done
+            by the operating system or network hardware. On getting, it
+            <em class="rfc2119" title="MUST">MUST</em> return the value of the
+            <code><a>OutgoingStream</a></code>'s <a>[[\WriteBufferedAmount]]</a> internal slot.
+          </dd>
+          <dt><code>writingAborted</code> of type <span class="idlAttriType"><a>StreamAbortInfo</a>
+          readonly</dt>
+          <dd>
+            <p>The <dfn id="dom-outgoingstream-writingAborted"><code>writingAborted</code></dfn>
+            attribute represents a promise that <a>resolves</a> when the
+            a message from the remote side aborting the stream is received.
+            For QUIC, that message is a STOP_SENDING frame.
+            When the <var>stream</var> receives this mesage, the <a>user agent</a>
+            <em class="rfc2119" title="MUST">MUST</em> run the following:
+            <ol>
+              <li>Let <var>stream</var> be the <code><a>OutgoingStream</a></code> object.
+              <li>Set <var>stream</var>'s <a>[[\Writable]]</a> slot to <code>false</code>.</li>
+              <li>Clear the <var>stream</var>'s write buffer.</li>
+              <li>Let <var>transport</var> be the <code><a>DataTransport</a></code>,
+              which the <var>stream</var> was created from.
+              <li>Remove the <var>stream</var> from the <var>transport</var>'s
+              <a>[[\OutgoingStreams]]</a> internal slot.</li>
+              <li><a>resolve</a> the promise with the resulting
+              <code><a>StreamAbortInfo</a></code> with the <code>errorCode</code>
+              set to the value from the aborting message from the remote side.</li>
+            </ol>
+          </dd>
+        </dl>
+      </section>
+      <section>
+        <h3>Methods</h3>
+        <dl data-link-for="OutgoingStream" data-dfn-for="OutgoingStream" class=
+        "methods">
+          <dt><dfn><code>write</code></dfn></dt>
+          <dd>
+            <p>Writes data to the stream. When the remote <code><a>DataTransport</a></code>
```

Comment:
Maybe it would be more clear to say it "adds data to the write buffer"?  I feel like that's the best summary of what happens, as well as to why this method returns void.

Reply:
Good idea.